blog-mysql-seed/mysql/mysql.star

```diff
@@ -0,0 +1,52 @@
+MYSQL_IMAGE = "mysql:8.0.32"
+
+def create_database(plan, database_name, database_user, database_password, seed_script_artifacts = []):
+    files = {}
+    for index, artifact in enumerate(seed_script_artifacts):
+        files["/docker-entrypoint-initdb.d/{}.sql".format(index)] = artifact
+    service_name = "mysql-{}".format(database_name)
+    mysql_service = plan.add_service(
+        service_name = service_name,
+        config = ServiceConfig(
+            image = MYSQL_IMAGE,
+            ports = {
+                "db": PortSpec(
+                    number = 8080,
```

Comment:
Is that an acceptable default port for mysql? I know postgreas is 5432, 8080 feels kind of to generic for a DB port, but trusting you